packages/storage/test/unit/requests.test.ts

```diff
@@ -347,6 +359,34 @@ describe('Firebase Storage > Requests', () => {
     const url = requestInfo.handler(fakeXhrIo({}), serverResourceString);
     assert.equal(url, downloadUrlFromServerResource);
   });
+
+  it('generateSignedURL request info', () => {
+    const maps: Array<[Location, string]> = [
+      [locationNormal, locationNormalUrl],
+      [locationEscapes, locationEscapesUrl]
+    ];
+    for (const [location, url] of maps) {
+      const requestInfo = generateSignedURL(storageService, location);
+      assertObjectIncludes(
+        {
+          url:
+            makeUrl(url, storageService.host, storageService._protocol) +
+            ':generateSignedUrl',
+          method: 'POST',
+          body: '{}',
+          headers: { 'Content-Type': 'application/json; charset=utf-8' }
+        },
+        requestInfo
+      );
+    }
+  });
+  it('generateSignedURL handler', () => {
+    const requestInfo = generateSignedURL(storageService, locationNormal);
+    const signedURLResourceString = `{ "signed_url": "${serverSignedURL}" }`;
```

Comment:
Is there a reason why this can't be 
```javascript
`{ "signed_url": ${serverSignedURL}}`;
```

Reply:
Yes. ${serverSignedURL} evaluates to a string but without the surrounding double quotes, leading to an invalid JSON object being created and a null url being returned. Confirmed this by testing with single, double, and no quotation marks. JSON.stringify() was not an option since the linter does not allow for objects with non-camelcase keys.